prep-javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,39 @@
+const number1 = 5;
+const number2 = 6;
+const product = number1 * number2;
+console.log(product);
+console.log(typeof product);
+
+const charge = 12;
+const payment = 7;
+const amountRemaining = charge - payment;
+console.log(amountRemaining);
+console.log(typeof amountRemaining);
+
+const tests = 9;
+const assignments = 10;
+const final = 8;
+const grade = (tests + assignments + final) / 3;
+console.log(grade);
+console.log(typeof grade);
+
+const firstName = 'Vanessa';
+const lastName = 'Viray';
+const fullName = `${firstName}${' '}${lastName}`;
```

Comment:
You actually don't need `${' '}` if you are going to do a template literal. One of the advantages of using template literals, is it allows us to write strings just like how we would sentences. So this can become
```
const fullName = `${firstName} ${lastName}`
```
Since there is a space between `firstName` and `lastName` it will create the space for you